test/e2e/utils_test.go

```diff
@@ -313,13 +315,65 @@ func checkInternalMemberClusterExists(name, namespace string) {
 func checkMemberClusterNamespaceIsDeleted(name string) {
 	Eventually(func(g Gomega) error {
 		var ns corev1.Namespace
-		if err := hubClient.Get(ctx, types.NamespacedName{Name: name}, &ns); !errors.IsNotFound(err) {
+		if err := hubClient.Get(ctx, types.NamespacedName{Name: name}, &ns); !k8serrors.IsNotFound(err) {
 			return fmt.Errorf("member cluster namespace %s still exists or an unexpected error occurred: %w", name, err)
 		}
 		return nil
 	}, eventuallyDuration, eventuallyInterval).Should(Succeed())
 }
 
+func setupNetworkingCRDs() {
```

Comment:
Hi Arvind! I don't particularly think it would be a good idea to install CRD at this point; it's better to do that when we set up the hub cluster.

Reply:
The networking CRDs should be installed using manifests retrieved from a tagged branch in the networking repo, rather than using a checked-in stale copy.